portfoliostyle.css

```diff
@@ -0,0 +1,310 @@
+* {
+	Box-sizing: border-box;
+	font-family: monaco;
+	
+}
+
+div {
+	display: flex;
+}
+
+body {
+	/*background-color: #aaabb8;*/
+	background-image: url("images/snow-3161134_1920.jpg");
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover;
+}
+
+nav {
+	display: flex;
+	background-color: #25274d;
+	color: #ffffff;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+	padding: 0 25px 0 25px;
```

Comment:
Can combine the top and bottom, right and left values if they are the same like they are here: e.g. padding 0 25px; should give the same result.

Reply:
Changed to your suggestion, worked well